apps/client/src/plugins/Coordinates/CoordinatesTransformRow.js

```diff
@@ -67,6 +81,110 @@ class CoordinatesTransformRow extends React.PureComponent {
     });
   }
 
+  getCoordinates(coordinateFormatTitle) {
+    let inputX, inputY;
```

Comment:
Sorry but we really need a diffrent solution for this as the coordinate systems are fully customizable in the admin interface.  The can be one coordinatesystem or ten or whatever, and perhaps not SWEREF at all.

Reply:
This is correct. The logic for selecting the correct numeric formats x and y value is now updated.